public/static/pathways/her2_pathway.json

```diff
@@ -26,7 +26,8 @@
            "id": "1",
            "transition": "NodeStatus"
          }
-       ]
+       ],
+       "type": "other"
```

Comment:
This should probably be changed to `"start"` now, along with any remaining `"other"`s